server/redux/maps/map.js

```diff
@@ -18,20 +18,18 @@ let Maps = [ // 1  2  3  4  5  6  7  8  9  10 11 12
 
 /* RANDOMIZES DESTROYABLE BOXES & ENSURES SPAWN POSITIONS ARE CLEAR */
 const randomGeneration = (arr) => {
-  let newArr = [[],[],[],[],[],[],[],[],[],[],[],[],[],[], []]
-  for (let j = 0; j < arr.length; j++) {
-    for (let k = 0; k < arr[0].length; k++) {
-      if (arr[j][k] === 3) {
-        let random = Math.random();
-        if (random > 0.4) newArr[j].push(0)
-        else newArr[j].push(3)
+  return arr.map(row => {
+    return row.map(coord => {
+      if (coord === 3) {
+        const random = Math.random();
+        if (random > 0.4) { return 0; }
```

Comment:
Why not `if (Math.random() > 0.4)`? 

Also why hard code .4 ? Wouldn't you want this to be dynamic so you can modify this on the fly? Possibly from a config file?